src/erasure-code/CMakeLists.txt

```diff
@@ -3,9 +3,9 @@
 set(erasure_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/erasure-code)
 
 #jerasure subdir must be before shec so jerasure & neon obj libs are declared
-include_directories(jerasure/jerasure/include)
-include_directories(jerasure/gf-complete/include)
-include_directories(jerasure)
+include_directories(SYSTEM jerasure/jerasure/include)
+include_directories(SYSTEM jerasure/gf-complete/include)
+include_directories(SYSTEM jerasure)
```

Comment:
@adamemerson the header files under `src/erasure-code/jerasure` are not system headers, and they belong to Ceph project. so should not `SYSTEM` them, right?

Reply:
jerasure shouldn't be.

But jerasure/jerasure/ and jerasure/gf-complete are submodules that originated from an external project. I don't know how 'closely' we own them. I had thought we didn't own them closely at all and just had them because BOGUS LEGAL ACTIONS by horrible people in the past had made the originals disappear. If we're actually maintaining them pretty regularly I'll take them out, too.

Reply:
neither do i know. but Loïc has been the maintainer of these two project. see http://jerasure.org. but they are not actively updated, probably because they are both mature enough.

Reply:
All right. In that case I'll pull out the line for jerasure but not the other two. Since we have NEW warnings showing up just because of new compiler versions there isn't much reason to see them in something that isn't actively developed.